ui/src/org.ts

```diff
@@ -62,6 +62,44 @@ const orgFactoryAddress = (network: ethereum.Environment): string => {
   }
 };
 
+const ORG_POLL_INTERVAL_MS = 2000;
+
+// TODO use `ui/src/sleep` once
+// https://github.com/radicle-dev/radicle-upstream/pull/1985 is merged.
+const sleep = (timeMs: number): Promise<void> => {
+  return new Promise(resolve => {
+    setTimeout(resolve, timeMs);
+  });
+};
+
+// Update the org data for the sidebar store every
+// `ORG_POLL_INTERVAL_MS` milliseconds.
+const updateOrgsForever = async (): Promise<never> => {
+  for (;;) {
+    const walletStore = svelteStore.get(wallet.store);
+    const w = svelteStore.get(walletStore);
+
+    if (w.status === wallet.Status.Connected) {
+      await fetchOrgs();
+    }
+    await sleep(ORG_POLL_INTERVAL_MS).catch(err => {
```

Comment:
Will this flood the app with error notifications if we're offline?

Reply:
Good point. I’ve updated the code so that we only show the first error. (The `catch` was also in the wrong place).